python/CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 [0.5.4] - 2022-XX-XX
 --------------------
 
+**Features**
+ - The ``Tree`` object now has the method ``siblings`` to get
+   the siblings of a node. It returns an empty tuple if the node
+   has no siblings, is not a node in the tree, is a virtual root,
```

Comment:
Seems nitpicky, but I think it is important to conceptualise the virtual root as singular.
```suggestion
   has no siblings, is not a node in the tree, is the virtual root,
```

Reply:
Fixed